policies/rudderc/src/technique.schema.json

```diff
@@ -68,11 +71,10 @@
     },
     "reportingMode" :{
       "oneOf": [
-        { "const": "disabled" },
         {
           "type": "object",
           "properties": {
-            "mode": { "enum": [ "worst-case-weighted-one", "worst-case-weighted-sum" ]  }
+            "mode": { "enum": [ "worst-case-weighted-one", "worst-case-weighted-sum", "disabled", "enabled", "weighted" ]  }
```

Comment:
enabled is not accepted here

Reply:
we could easily accept it as an alias for worst-case-weighted-sum

Reply:
"banco"

Reply:
c'est déjà le cas je raconte nimp, heureusement que je ne suis qu'un contributeur du dimanche et pas un vrai dev

Reply:
Dans la doc tu dis que c'est ;) WorstReportWeightedSum  mais le code mais weighted en default !!

Reply:
Ha enabled est accepté ?

Reply:
yes lol